src/pages/FolderPage/components/ActionButton/ActionButton.tsx

```diff
@@ -3,14 +3,32 @@ import styles from "./actionButton.module.css";
 import shareIcon from "assets/images/ic_share.svg";
 import penIcon from "assets/images/ic_pen.svg";
 import trashIcon from "assets/images/ic_trash.svg";
+import { OpenModal } from "utils/hooks/useModal";
 
-function ActionButton({ openModal, variant }) {
-  const variantInfo = {
-    shared: { icon: "공유", name: "공유", imgUrl: shareIcon },
-    edit: { icon: "펜", name: "이름변경", imgUrl: penIcon },
-    "delete-folder": { icon: "쓰레기통", name: "삭제", imgUrl: trashIcon },
+interface Props {
+  openModal: OpenModal;
```

Comment:
이렇게 작성해주는게 더 명확할 것 같네요!
```suggestion
  openModal: (name: string) => void;
```